webpack.config.dev.js

```diff
@@ -0,0 +1,100 @@
+const path = require('path');
```

Comment:
We use the `dev-webpack.config.js` naming convention in other projects should probably use it here